_episodes/03-working-with-MARC-files.md

```diff
@@ -21,10 +21,91 @@ keypoints:
 ---
 
 
-# Working with MARC files
+##Working with MARC files
 
+MarcEdit recognizes the following MARC file types:
 
+<table>
+  <tr>
+   <td><strong>File type</strong>
+   </td>
+   <td><strong>File extension</strong>
+   </td>
+   <td><strong>Usage</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Binary MARC file
+   </td>
+   <td>mrc
+   </td>
+   </td>
+   <td>File format typically used in an  ILS or LSP. Other file extensions provided by vendors (ex. marc, dat, bin) are equivalent.
+   </td>
+  </tr>
+  <tr>
+   <td>Mnemonic MARC Text File
+   </td>
+   <td>mrk
+   </td>
+   <td>File format used by MarcEdit.
+   </td>
+  </tr>
+  <tr>
+   <td>MARC UTF-8 Text File
+   </td>
+   <td>mrk8
+   </td>
+   </td>
+   <td>Legacy file format for MARC mnemonic files saved with UTF8 encoding.
+   </td>
+  </tr>
+  <tr>
+   <td>MARCXML file
+   </td>
+   <td>xml
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>MODS file
+   </td>
+   <td>xml
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Text files
+   </td>
+   <td>txt
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
 
-**1. Getting Started**
+However, to work with a MARC file in the MARCEditor your file needs to be in MARC mnemonic format.
```

Comment:
Each line to be included in the callout should start with ">"
There needs to be a heading for the callout - e.g.
```
>## Heading
>Callout content 1
>Callout content 2
{: .callout}
```